pwndbg/commands/__init__.py

```diff
@@ -444,6 +444,8 @@ def __init__(self, parser_or_desc, aliases=[], command_name=None):
         # We want to run all integer and otherwise-unspecified arguments
         # through fix() so that GDB parses it.
         for action in self.parser._actions:
+            if isinstance(action, argparse._SubParsersAction):
+                action.type = str
```

Comment:
@disconnect3d it seems like this is the first command trying to use a subparser. I needed to add this otherwise the subcommand would be parsed as a `gdb.Value` instead of a string. I'm not very familiar with `ArgparsedCommand` so I'm not sure if this is the right solution, just FYI.

Reply:
I am not familiar with subparsers, but it would be nice to make our argparsing and handling of int vs gdb.Value vs multiple strings clear and documented 😅